src/test/rpc_tests.cpp

```diff
@@ -339,6 +339,29 @@ BOOST_AUTO_TEST_CASE(rpc_convert_values_generatetoaddress)
     BOOST_CHECK_EQUAL(result[2].get_int(), 9);
 }
 
+BOOST_AUTO_TEST_CASE(rpc_convert_named_values_generatetoaddress)
+{
+    UniValue result;
+    std::vector<std::string> valid_named_values = {
+        "nblocks=10",
+        "address=2N7xz1EnWubTbQmWFcVpCvT2RwKeDuzWr5g",
+        "maxtries=50"
+    };
+
+    BOOST_CHECK_NO_THROW(result = RPCConvertNamedValues("generatetoaddress", valid_named_values));
+    BOOST_CHECK_EQUAL(result[0].get_int(), 10);
+    BOOST_CHECK_EQUAL(result[1].get_str(), "2N7xz1EnWubTbQmWFcVpCvT2RwKeDuzWr5g");
+    BOOST_CHECK_EQUAL(result[2].get_int(), 50);
+
+    std::vector<std::string> invalid_named_values = {
```

Comment:
what do you think about separating out the invalid value test logic into an entirely new test? I personally believe tests should be as small and focused as possible - would love to know what you think.

Reply:
I followed the other tests from rpc_tests.cpp when writing this one. Not sure if this function should be tested via two separate tests. Nothing agains more tests, of course, so feel free to provide alternatives.

Reply:
Hmm, I'm new to the codebase, I guess we should stick to the style that already exists.